### PR TITLE
Add support for themes in Quill

### PR DESCRIPTION
**Pull request by JoshuaSP**

### CODE REVIEW

File: quill/quill.d.ts [hunk 1]
```diff
@@ -1,3 +1,8 @@
+// Type definitions for quill.js
+// Project: https://github.com/quilljs/quill/tree/0.20.1
+// Definitions by: Mike Morearty <https://github.com/mmorearty>
```
Comment (mmorearty) [inline quill/quill.d.ts]: These definitions are definitely not by me. Please change this. Thanks!

Comment (JoshuaSP) [inline quill/quill.d.ts]: Do you know the set of circumstances by which you are git blamed for all the code and how I might find the actual author?

Comment (mmorearty) [inline quill/quill.d.ts]: All I did was to move it from our private codebase over to DefinitelyTyped. Please see my private email to you. Thanks!

**Comment by JoshuaSP** [comment]
Fixed the attribution.

File: quill/quill.d.ts [hunk 2]
```diff
@@ -1,3 +1,8 @@
+// Type definitions for quill.js
+// Project: https://github.com/quilljs/quill/tree/0.20.1
+// Definitions by: Yao Yujian <https://github.com/yyjhao>
```
Comment (epelz) [inline quill/quill.d.ts]: We generally use `// Definitions by: Asana <https://asana.com>` for the DefinitelyTyped attributions.

**Comment by JoshuaSP** [comment]
Sounds good, done!

**Comment by JoshuaSP** [comment]
I have, thanks!

**Pull request merged**